src/main/java/taxi/controller/user/LoginController.java

```diff
@@ -0,0 +1,41 @@
+package taxi.controller.user;
```

Comment:
```suggestion
package taxi.controller.authentication;
```
I think it's a better name for a package